docs/aws_kms.md

```diff
@@ -36,4 +41,86 @@ region | string | ✅ | Region where key is created
 
 The fields `access_key_id` & `secret_access_key` can be set in the environment variables `AWS_ACCESS_KEY_ID` & `AWS_SECRET_ACCESS_KEY` respectively.
 
-Import command is not available for aws as there is no support for asymmetric keys in AWS KMS. (Ref: <https://docs.aws.amazon.com/kms/latest/developerguide/importing-keys.html>)
+## Importing a key into AWS
+
+The Import command is not available for aws as there is no support for asymmetric keys in AWS KMS. (Ref: <https://docs.aws.amazon.com/kms/latest/developerguide/importing-keys.html>) 
+
+However, you can generate a private key in an external environment and then import it into AWS using this [guide](https://docs.aws.amazon.com/kms/latest/developerguide/importing-keys.html)
+
+## Key generation in AWS

```

Comment:
This part is bound to change often, so it is better for users to refer to an AWS document rather than a static mention here in signatory.